modules/download_to_pc.R

```diff
@@ -33,11 +33,13 @@ download_pc_button_server <- function(id,
         # geef de filename op, zou via interactieve kunnen
         filename = function(){
           # Create file name
-          paste0('data_', name_munproj(),'_', selected_start_date(),
+          f <- paste0('data_', name_munproj(),'_', selected_start_date(),
                  '_', selected_end_date(), '.zip')
+          log_trace("download_pc_button: filename set at {f}")
```

Comment:
Could you add the name of the module?
log_trace("mod dwnld pc: download_pc_button: filename set at {f}")